pkg/volume/host_path/host_path.go

```diff
@@ -103,6 +104,12 @@ func (plugin *hostPathPlugin) NewMounter(spec *volume.Spec, pod *v1.Pod, _ volum
 	if err != nil {
 		return nil, err
 	}
+
+	err = validation.ValidatePathNoBacksteps(hostPathVolumeSource.Path)
```

Comment:
@kubernetes/sig-storage-pr-reviews is the appropriate place to error on an invalid path here or in SetUp?